app/lib/reducers/ui.js

```diff
@@ -111,7 +111,7 @@ const reducer = (state = initial, action) => {
         }
 
         if (null != config.colors) {
-          if (state.colors.toString() !== config.colors.toString()) {
+          if (JSON.stringify(state.colors) !== JSON.stringify(config.colors)) {
```

Comment:
Ah yeah, ok I think I understand now. So you're concern is that if they have an array, but we're checking for an object, it will always be different, right? The check here will work, but if they have an array in their config still, it will always run the block inside here, since they will never be the same. If that's the case, I can add some logic right after work today to check if it's an array or an object. Let me know if I'm not reading that right 😄